<ui version="4.0" >
 <class>SinerjiGui</class>
 <widget class="QDialog" name="SinerjiGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sinerji</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>&lt;b>Yönetmek istediğiniz bilgisayarı ve fiziksel konumunu seçin&lt;/b></string>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="1" >
       <widget class="QComboBox" name="topComboBox" />
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="leftComboBox" />
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="rightComboBox" />
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="bottomComboBox" />
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="iconLabel" >
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap>images/icon.png</pixmap>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>189</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>images/buttonOk.png</normaloff>images/buttonOk.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>images/buttonCancel.png</normaloff>images/buttonCancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
